From 26738fbf5b08a8e6b51cffae1bc42cef36d8465a Mon Sep 17 00:00:00 2001
From: Alistair Delva <adelva@google.com>
Date: Mon, 7 Feb 2022 09:45:07 -0800
Subject: [PATCH] Revert "clk: Emit a stern warning with writable debugfs
 enabled"

This reverts commit 489a71964f9d74e697a12cd0ace20ed829eb1f93.

Rationale does not apply to Android userspace which does not allow
access to debugfs on user builds.

Bug: 218319878
Signed-off-by: Alistair Delva <adelva@google.com>
Change-Id: Ia36b04b432a3db330e5a12763a3250869b97f0fe
Signed-off-by: Steve Muckle <smuckle@google.com>
Signed-off-by: Lee Jones <joneslee@google.com>
---
 drivers/clk/clk.c | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/drivers/clk/clk.c b/drivers/clk/clk.c
index ae07685c7588..8d4568e8e2ff 100644
--- a/drivers/clk/clk.c
+++ b/drivers/clk/clk.c
@@ -3603,24 +3603,6 @@ static int __init clk_debug_init(void)
 {
 	struct clk_core *core;
 
-#ifdef CLOCK_ALLOW_WRITE_DEBUGFS
-	pr_warn("\n");
-	pr_warn("********************************************************************\n");
-	pr_warn("**     NOTICE NOTICE NOTICE NOTICE NOTICE NOTICE NOTICE           **\n");
-	pr_warn("**                                                                **\n");
-	pr_warn("**  WRITEABLE clk DebugFS SUPPORT HAS BEEN ENABLED IN THIS KERNEL **\n");
-	pr_warn("**                                                                **\n");
-	pr_warn("** This means that this kernel is built to expose clk operations  **\n");
-	pr_warn("** such as parent or rate setting, enabling, disabling, etc.      **\n");
-	pr_warn("** to userspace, which may compromise security on your system.    **\n");
-	pr_warn("**                                                                **\n");
-	pr_warn("** If you see this message and you are not debugging the          **\n");
-	pr_warn("** kernel, report this immediately to your vendor!                **\n");
-	pr_warn("**                                                                **\n");
-	pr_warn("**     NOTICE NOTICE NOTICE NOTICE NOTICE NOTICE NOTICE           **\n");
-	pr_warn("********************************************************************\n");
-#endif
-
 	rootdir = debugfs_create_dir("clk", NULL);
 
 	debugfs_create_file("clk_summary", 0444, rootdir, &all_lists,
-- 
2.34.1

